ansible/roles/common/tasks/main.yml

```diff
@@ -135,11 +136,35 @@
     state: directory
   with_items:
     - /var/lib/liquid
+    - /var/lib/liquid/data
+    - /var/lib/liquid/data/postgresql
     - /var/lib/liquid/conf
     - /var/lib/liquid/https
     - /var/lib/liquid/https/nginx
     - /var/lib/liquid/https/certs
 
+- name: Write postgresql configuration file
+  copy:
+    src: postgresql.conf
+    dest: /etc/postgresql/9.5/main/postgresql.conf
+
+- name: Move postgresql data
+  shell: mv /var/lib/postgresql/9.5 /var/lib/liquid/data/postgresql/
+
+- name: Move rabbitmq data
+  shell: mv /var/lib/rabbitmq /var/lib/liquid/data/
+
+- name: Link rabbitmq data to new location
+  shell: ln -s /var/lib/liquid/data/rabbitmq /var/lib/rabbitmq
```

Comment:
This task can't be rerun. Maybe use `creates: /var/lib/rabbitmq`? Same for the two tasks above.

Reply:
will do